airbyte-integrations/connectors/source-stripe/source_stripe/components.py

```diff
@@ -0,0 +1,21 @@
+from typing import Iterable, Mapping, Any
+
+import requests
+from airbyte_cdk import RecordExtractor
+
+
+class StripeDualStreamRecordExtractor(RecordExtractor):
+    def extract_records(self, response: requests.Response) -> Iterable[Mapping[str, Any]]:
+        if response.json()["url"] == "/v1/events":
+            for record in response.json()["data"]:
+                item = record["data"]["object"]
+                item["updated"] = record["created"]  # FIXME move to transformation
+                item["pagination_id"] = record["id"]  # FIXME move to transformation
```

Comment:
I'm not actually sure if we can flatten this up using a transformation but at least, it is possible to do as part of an extractor (although very unconventional)